gokart/slog_config.py

```diff
@@ -0,0 +1,84 @@
+import functools
+import logging
+import os
+
+from pythonjsonlogger import json
+
+
+class SlogConfig(object):
+    """
+    SlogConfig is for logging configuration, Utility-class.
+    This class will read an environment variable named GOKART_LOGGER_FORMAT, then switch logging configuration.
+
+    If "$GOKART_LOGGER_FORMAT=json", set logging configuration as structured logging.
+    Else if "$GOKART_LOGGER_FORMAT=text", set logging configuration as plain text, normal logging.
+    Otherwise, default setting is applied, structured logging.
+    """
+
+    default_base_format = '%(message)s %(lineno)d %(pathname)s %(asctime)s %(name)s %(levelname)s %(filename)s %(lineno)s %(message)s'
+    default_date_format = '%Y/%m/%d %H:%M:%S'
+
+    @staticmethod
+    def switch_log_format(logger):
+        """
+        If $GOKART_LOGGER_FORMAT=json, set logging configuration as structured logging.
+        On the other hand, set logging configuration as plain text.
+        """
+        logger_mode = os.environ.get('GOKART_LOGGER_FORMAT')
+        if logger_mode and logger_mode.lower() not in {'text', 'json'}:
+            raise Exception(f'Unknown logger format: {logger_mode}')
+        # plain text mode, so nothing is applied,
+        elif logger_mode and logger_mode.lower() == 'text':
+            return logger
+        else:
+            if not isinstance(logger, logging.Logger):
+                return logger
```

Comment:
we can reduce this nested codes
```suggestion
        if logger_mode and logger_mode.lower() not in {'text', 'json'}:
            raise Exception(f'Unknown logger format: {logger_mode}')
        # plain text mode, so nothing is applied,
        if logger_mode and logger_mode.lower() == 'text':
            return logger
        if not isinstance(logger, logging.Logger):
            return logger
```